includes/agentic-commerce/class-wc-stripe-agentic-commerce-price-availability-responder.php

```
@@ -0,0 +1,181 @@
+		$response['availability'] = $this->get_availability( $product );
+
+		$currency = get_woocommerce_currency();
+
+		// The feed publishes the regular price as `price` and the sale price
+		// separately; mirror that split so the real-time answer matches the
+		// catalog. Delegated checkout applies no WooCommerce cart discounts,
+		// so catalog prices are the authoritative selling prices here.
+		$regular_price = $product->get_regular_price();
+		if ( '' !== $regular_price ) {
+			$response['price'] = [
+				'unit_amount' => WC_Stripe_Helper::get_stripe_amount( (float) $regular_price, $currency ),
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'respond'. (undefined)

(StaticAccess)